### ISSUE TITLE
znt_list_instant_select is not working after some time

### ISSUE THREAD

**Issue by seqizz** [psprint/zsh-navigation-tools#27]
First of all, great plugin.

I've used the plugin via zplug and I'm using `znt_list_instant_select=1` in my zshrc and it is working. But after a while I am getting the old behavior again. The whole settings seems to change (borders are back etc) and I couldn't find what triggers this. Maybe I am pressing ctrl-C too often :|

If I re-source `~/.zshrc` or open a new shell, it is ok.

Is there a cache somewhere, expiring etc?

**Comment by psprint** [comment]
I looked at it and it seems to not be easy. Could a small donation be granted for this? Like $4 or $5? I would approach this today. My paypal email is sgniazdowski AT gmail.com.

**Comment by seqizz** [comment]
Oh wow, this was the most direct approach I've ever seen. Not to say it's a bad thing, but personally I don't think this issue bothers me enough to pay for it. I've been using it for 2 days and just wanted to report.

Since I respect your labor, I've removed the plugin and switched to [zsh-peco-history](https://github.com/jimeh/zsh-peco-history) instead.

Thanks for the plugin anyway.

**Comment by psprint** [comment]
Sorry, I'm doing 7.5 commits per day not excluding holidays (2017 Github stats; ~4.5 for this year) and I've discovered that it helps to keep that level by using small dotations. Sorry if I was too direct.

**Comment by psprint** [comment]
I've switched on "Contributions overview" on my Github profile. It looks like this:

![contribution_overview](https://user-images.githubusercontent.com/6049288/48699724-6558e780-ebea-11e8-89a8-160b3344fc2c.png)

I've did 1565 commits in last 12 months, that's 4.3 commits per day. I just want to say that committing each day in a year 4 times is something like being hired by an open source company or normal company to work on open source projects. So I've started to treat this donations matter quite direct, feeling like "is there a job for me to do?", not "what could I code today to relax and exercise mind?".
